src/UglyToad.PdfPig/Util/ColorSpaceDetailsParser.cs

```diff
@@ -56,7 +55,7 @@ internal static class ColorSpaceDetailsParser
                 var colorSpaceDetails = GetColorSpaceDetails(colorSpace, imageDictionary.Without(NameToken.Filter).Without(NameToken.F), scanner, resourceStore, filterProvider, true);
 
                 var decodeRaw = imageDictionary.GetObjectOrDefault(NameToken.Decode, NameToken.D) as ArrayToken ?? new ArrayToken([]);
-                var decode = decodeRaw.Data.OfType<NumericToken>().Select(x => x.Double).ToArray();
+                var decode = decodeRaw.Data.OfType<NumericToken>().Select(static x => x.Double).ToArray();

```

Comment:
what does `static` means in this context?

Reply:
This ensures that the inner method doesn't capture (or allocate a closure). It also allows you to reuse variable names inside the static scope. 

The code gen in this case is the same, but these are nice to make sure we don't accidentally capture inside the lambda.